### ISSUE TITLE
How to make an element (a grid?) invisible

### ISSUE THREAD

**Issue by WangWei90** [jipolanco/WriteVTK.jl#44]
Thank you for the efforts of publishing this package! It helps a lot.

 I want to know how to make some elements invisible by using VTK files, like set the value on those locations to NaN or nothing or something else.

Recently, I conducted some 3D-SEM tests, and I use WriteVTK to construct the 3D models. The process is very smooth. The problem is that the paraview eat huge memory (>30G) when I just use the threshold to remove some background color (some boundaries). So, I want to remove those background color directly in VTK files, what should I do?

**Comment by jipolanco** [comment]
I'm glad you find the package useful!

I have not dealt with this problem myself, so I am not sure if I can be of much help. One idea would be to set to NaN (from within Julia) the values of your dataset that you want to hide. Then in paraview you can assign a specific colour to NaN values in your dataset.

**Comment by WangWei90** [comment]
Thank you! It really helps.
I set the background color to NaN32 and it works as you said by using NaN color (opacity=0).
(The existences of NaN, NaN32, missing, nothing are really useful, I just discover these)

Before I set the background color to NaN
![screenshot_20181211_215633](https://user-images.githubusercontent.com/20700315/49805812-176e7400-fd91-11e8-90ea-8cc362bf1d5e.png)

After I set the background color to NaN
![screenshot_20181211_215651](https://user-images.githubusercontent.com/20700315/49805840-248b6300-fd91-11e8-82b8-fd1ce055e77e.png)

But, there is a problem existing
![screenshot_20181211_215721](https://user-images.githubusercontent.com/20700315/49805866-340aac00-fd91-11e8-9cc2-f83cf112fadb.png)

What I really want is like this (using Clip bottom)
![screenshot_20181211_220259](https://user-images.githubusercontent.com/20700315/49805888-41c03180-fd91-11e8-9c3f-4908421bbcd4.png)

This is model is a fraction of the total model. 
I can't use threshold, clip bottom in the total model because paraview will be stuck.
It seems the volume render will lose precision (blurred compared to surface render)?
